spec/requests/api/v0.1/trainees_spec.rb

```diff
@@ -40,6 +41,74 @@
     end
   end
 
+  describe "POST /api/v0.1/trainees" do
```

Comment:
Do you think we should split the different endpoints into different request spec files?